### PR TITLE
Add global keypress event handler to make the auto-focus to input message

### PR DESCRIPTION
**Pull request by nishimaki10**
@RocketChat/core 

Auto focus to `textarea.input-message` when not focused on any input
areas.

### CODE REVIEW

**Comment by sampaiodiego** [comment]
I like the idea, but I think it should be places somewhere else. Like on [main.coffee](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-ui-master/master/main.coffee)

File: packages/rocketchat-ui/lib/fireEvent.js [hunk 1]
```diff
@@ -33,3 +33,15 @@ window.addEventListener('message', (e) => {
 			break;
 	}
 });
+
+window.addEventListener('keydown', (e) => {
+	const target = e.target;
+	if (/input|textarea/i.test(target.tagName)) {
```
Comment (sampaiodiego) [inline packages/rocketchat-ui/lib/fireEvent.js]: Should it test select as well?

**Comment by edwardmuel** [comment]
This breaks keyboard navigation. I pasted the event into Chrome's console to try it out and, as I suspected, now I can't use up/down/pgup/pgdn/home/end to scroll the chat area or navbar when the message input box is visible.

**Comment by nishimaki10** [comment]
@edwardmuel Thank you for review! I changed `keydown` to `keypress`.
> The keypress event is fired when a key is pressed down and that key normally produces a character value
https://developer.mozilla.org/en-US/docs/Web/Events/keypress

**Pull request merged**